test/seo.test.ts

```diff
@@ -179,6 +179,46 @@ describe("init with default options", () => {
 				},
 			]);
 		});
+
+		it("a page does not overwrite the initial default config", () => {
+			let [meta, links] = getSeo({
+				title: "About us",
+				twitter: { card: "summary_large_image" },
+			});
```

Comment:
you'll notice that this actually breaks more than just this test, it also affects the status of unrelated tests since the defaultConfig value is mutated between runs!